components/organisms/TopNav/top-nav.tsx

```diff
@@ -11,14 +11,14 @@ import { useFetchUser } from "lib/hooks/useFetchUser";
 
 const TopNav: React.FC = () => (
   <header className="top-nav-container w-full fixed top-0 left-0 z-50 py-0.5 bg-light-slate-3 border-b">
-    <div className="flex justify-between items-center mx-auto container px-2 md:px-16">
```

Comment:
Are these style changes just prettier or something else reordering the same Tailwind classes?

Reply:
It is an extension from vs code that automatically reorders tailwind classes https://github.com/heybourn/headwind

Reply:
Can we make this part of our ljnting? Not everyone might have that extension so, it'll cause unnecessary changes.

Reply:
A quick Google led me to https://www.npmjs.com/package/eslint-plugin-tailwindcss

I'll open an issue so that we can discuss it.